.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "pycmake"]
```

Comment:
It does with the second commit in the series. It's trivial to postpone it to a separate PR though